packages/e2e-tests/puppeteer/helpers.ts

```diff
@@ -92,3 +105,17 @@ export async function setUpBrowser(headless: boolean, slowMo?: number): Promise<
 
   return browser;
 }
+
+export async function waitAndOpenChannel(page: Page): Promise<void> {
+  const createChannelButton =
+    'body > div:nth-child(3) > div > div > div:nth-child(2) > div > button';
```

Comment:
Should we make this more robust to future changes in the UI? I think elsewhere we have used id selectors, which might be better.

Reply:
done.